Scripts/assignment/assignment_period.py

```diff
@@ -743,7 +743,8 @@ def _calc_extra_wait_time(self):
                                   + b["ctime"]*cumulative_time
                                   + b["cspeed"]*cumulative_speed)
                 # Estimated waiting time addition caused by headway deviation
-                segment["@wait_time_dev"] = headway_sd**2 / (2.0*line.headway)
+                headway_attr = self.extra("hw")
```

Comment:
Consider lifting definition of `headway_attr` out of `segment` loop.

Reply:
Thanks, now fixed!